experiments/dataSources/worldBank/readme.md

```diff
@@ -0,0 +1,102 @@
+# Analyse World Bank statictical data in Skuid
+The World Bank provides free and open access to global development data.  Information like GDP, Population, Economic growth and Development measures are available for your Skuid pages. This is one of the easiest API to interact with. 
+
+Some introductory links with information about this API from the World Bank Website: 
+-  [Overview of the data available through this API](https://datahelpdesk.worldbank.org/knowledgebase/articles/889386-developer-information-overview) 
+
+- [About the API:](https://datahelpdesk.worldbank.org/knowledgebase/articles/889392)
+
+- [The Basic Call Structure:](https://datahelpdesk.worldbank.org/knowledgebase/articles/898581)
+
+
+This API is exposed without any authentication - which makes it super easy to use in Skuid. It is a great way to explore the following concepts in Skuid:
+- Searching / filtering for a particular record.
+- Using stastical data in dashboards. 
+
+## How to use this data source
+
+### Create the data source
+
+First you'll need to create a new REST data source. 
+* Type: REST
+* Name: WorldBank
+* URL / Endpoint: ``http://api.worldbank.org/v2/``
+* Use Proxy: False
+* Authentication Method: No Authentication
+
+
+#### Double check the remote site setting
+
+* If you are using Skuid on Salesforce, you will be prompted to appove the creation of a remote site setting for this data source. If your pages do not work, you might need to edit it.  Check in Salesforce Setup at **Security > Remote site settings**.   
+
```

Comment:
```suggestion
* If you are using Skuid on Salesforce, you will be prompted to appove the creation of a remote site setting for this data source. If your pages do not work, you might need to edit it.  Check in Salesforce Setup at **Security > Remote site settings**.  

If the remote site setting is not present, create a new one and enter these values:

Remote Site Name: WorldBank
Remote Site URL: http://api.worldbank.org
```

Reply:
We should note what the values of this remote site setting should be. Skuid didn't create mine when I made the data source.